src/pages/logosValidator/dashboardDefinition.ts

```diff
@@ -19,682 +20,707 @@ export type LogoDefinition = {
    * A URL of the definition of the logo
    */
   link?: string;
+  type: InsightType;
   /**
    * To specify if the game needs to be restricted to a specific robotoff predictor
    * @default "universal-logo-detector"
    */
   predictor?: string;
 };
-
```

Comment:
interface Logos {
  [key: string]: LogoDefinition;
}

Reply:
The is that latter on the keys are used to let Typesctip detect speeling mistake and add autocompletion on VSCode
```
logos: (keyof typeof LOGOS)[];
```

With your solution key type is a string, so it breaks this check. I added a commit with the following solution:

```ts
type LogoType = { [key in keyof typeof UNTYPED_LOGOS]: LogoDefinition };

export const LOGOS = UNTYPED_LOGOS as LogoType;
```